### DISCUSSION TITLE
Unable to change pools

### DISCUSSION THREAD

**Discussion by glukkon24** [Chia-Network/chia-blockchain#7265] [Support]
I joined a pool before launch to see how things worked now that the pool I want to join is up and running, I have tried all day to change pools but clicking on change pools button does not work. Is this a glitch or is this not functioning as of yet or properly.

**Comment by loppefaaret** [comment]
without knowing what dev builds you have used on mainnet, and when you used them, and when you created plot NFT and plots, it is a bit hard to help you here... one option is to create a new plot NFT, and start plotting to that

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
